core/tests/protractor_desktop/topicAndStoryViewer.js

```diff
@@ -155,6 +160,21 @@ describe('Story viewer functionality', function() {
     await topicAndStoryViewerPage.expectUncompletedLessonCountToBe(1);
   });
 
+  it('should show description of a topic.', async function() {
+    await topicViewerPage.get('math', 'topic-tasv-one');
+    await topicViewerPage.expectedTopicInformationToBe('Description');
+  });
+
+  it('should show all published stories.', async function() {
+    await topicViewerPage.get('math', 'topic-tasv-one');
+    await topicViewerPage.expectedStoryCountToBe(1);
+  });
+
+  it('should show list of revision cards.', async function() {
+    await subTopicViewerPage.get('math', 'topic-tasv-one');
+    await subTopicViewerPage.expectedRevisionCardCountToBe(1);
+  });
```

Comment:
Can you combine these into one `it` block? Once we load the viewer page, we might as well check all of these at once

Reply:
Done.